src/module/item/armor/document.ts

```diff
@@ -92,9 +92,18 @@ class ArmorPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends Phy
         const maxPropertySlots = getPropertyRuneSlots(this);
         this.system.runes.property.length = Math.min(this.system.runes.property.length, maxPropertySlots);
 
-        // Add traits from fundamental runes
         const abpEnabled = ABP.isEnabled(this.actor);
         const baseTraits = this.system.traits.value;
+
+        // Determine if this armor uses runes or CTASEUP and clear the opposing data
+        const isSF2e = baseTraits.some((v) => v === "tech" || v === "analog");
```

Comment:
```suggestion
        const isSF2e = baseTraits.some((v) => ["tech", "analog"].includes(v));
```
It really is magical